tests/ExamplesTest.php

```diff
@@ -304,7 +308,14 @@ public function testWithTransaction(): void
 
     private function assertExampleOutput(string $file, string $expectedOutput): void
     {
-        require $file;
+        // Hide deprecation messages
+        $level = error_reporting(E_ALL ^ E_DEPRECATED);
```

Comment:
See [PHPLIB-1529](https://jira.mongodb.org/browse/PHPLIB-1529)

Reply:
Ah, thanks for filing this. Looks like this could've been removed when I dropped support for MongoDB 3.6: https://github.com/mongodb/mongo-php-library/commit/de3aaf881dcf8a4807529bf9b1ad4d609d101107#diff-97711093784103ece28740e725cc6a5f44bba1a469d5dcb78ef94a85778b3549L473

I'm a bit surprised that we don't have a sniff that alerts us to unused arguments 🤔

Reply:
`psalm --find-dead-code` returns only false positive.